packages/react/src/components/heliopay-container/index.tsx

```diff
@@ -124,7 +165,66 @@ const HelioPayContainer: FC<HeliopayContainerProps> = ({
     return currencyList.find((c: any) => c.symbol === currency);
   };
 
-  const handleSuccessPayment = (event: SuccessPaymentEvent) => {
+  useEffect(() => {
+    if (
+      paymentDetails?.features?.requireDeliveryAddress &&
+      paymentDetails?.features?.requireCountry
+    ) {
+      getCountry();
+    }
+
+    if (
+      paymentDetails?.currency != null &&
+      paymentDetails?.normalizedPrice != null
+    ) {
+      if (
+        paymentDetails?.fixedCurrency &&
+        paymentDetails?.features?.requireFixedCurrency
```

Comment:
Do we even have requireFixedCurrency on dynamic links features object? I think you should use the prop passed through